start/sw/web-app/laravel5.md

```diff
@@ -9,46 +9,53 @@ order: 4
 
 Laravel is a high-level PHP Web framework.
 
-#### Install Apache Web Server
-Follow the Apache installation instructions [here](/start/sw/web-app/apache.html).
+#### Install Nginx Web Server
+Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
 
-#### Install MariaDB Relational Database Server
-Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
-
-#### Install PHP
-Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
-#### Install PHP Extensions
+#### Install MariaDB Relational Database Server (Or other database you prefer)
+Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
 
+Or you can install by : 
 ```bash
-# Laravel required PHP extensions.
-$ sudo dnf install php php-common php-cli php-pdo php-mbstring php-zip php-xml
-# Restart Apache
-$ sudo systemctl restart httpd
+# install MariaDB
+$ dnf install mariadb-server mariadb
+# start MariaDB
+$ systemctl enable --now mysql
+# initialize MariaDB
+$ sudo mysql_secure_installation
 ```
 
-#### Install Composer
-We will need composer to install laravel and it's dependencies
+#### Install PHP and all its dependencies
+Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
+OR, you can rip the benefit of using Fedora, and install composer directly instead. This will install PHP, and all the required extensions/libraries needed with dnf (And of course, updates with dnf).
```

Comment:
In that case, I'd say you can just state that php will be pulled in with composer and there's no need to follow the instructions (just point to them in case user needs them for some reason).

Reply:
Also please omit the part about updates(specifically `needed with dnf (And of course, updates with dnf).`). We try to keep it simple.